code/src/Form components/Name.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export const Name = ({name, setName}) => {
+
+    return (
+        <fieldset className="container">
+            <legend tabIndex="0">Please enter your name</legend>
+            <label for="name">
```

Comment:
In React files we need to use `htmlFor` instead of normal `for` attribute - it's because `for` is a reserved keyword in JavaScript and we need to remember that JSX is actually normal JS, not HTML ;)